proxy_config_spec.yaml

```diff
@@ -0,0 +1,26 @@
+---
+debug: true
```

Comment:
It looks like this is identical to `example/veneur-proxy/config.yaml`, in which case we should consolidate these into a single file. The motivation behind having the example generate the config is that we don't have to ensure separately that our documentation (the example) stays in sync with our code, because any drift becomes a build error.

(We accidentally broke that invariant - mea culpa! - but we should restore that here!)

Reply:
I think what I was missing in this original change was that a value being missing (i.e. commented out) is no different from it being present but with a zero value. With that oversight corrected, the only thing I could imagine is type hinting. For example a field that needs a float versus an int?

That seems a small concern though, so I think I am in agreement with the above and that we should consolidate!

Reply:
`proxy_config_spec.yaml` is *very* much not the same as `config_spec.yaml` - see the `consul_` prefixed keys for an example. These two configs aren't compatible with each other and I don't think it's in scope for this PR to make them compatible

Reply:
Oh, i misread that this was proxy. That one is definitely different. :)

Reply:
(oh damn, misread your comment, deleted mine)

Yep, I agree, we should probably put in symlinks from the example dir to the _spec files.